### DISCUSSION TITLE
Synology nas @eadir directory being scanned any way to stop it?

### DISCUSSION THREAD

**Discussion by firstWizball** [LMS-Community/slimserver#981] [Q&A]
Hi the Synology nas puts a directory called @eaDir in to all the music directories when you use their media server.
I have looked in my pref files and I cannot see anything that says ignoredir in it.
I was wondering if anyone can help set up the file o not scan for these directories.

Many thanks
Peter

**Comment by terual** [comment]
I think this is an permission issue, because I do not see eadir in LMS running on Synology.

Can you provide some more information? Is LMS running on the NAS? Are you using docker? Which image? What are the docker options? How are your permissions setup on the music files?

**Comment by firstWizball** [comment]
I am running a Ubuntu server VM version 20
Have mounted the folder to the VM
I have the media and audio station installed.
So I have both the audio station and lms pointing to the same folder.

Anything else you need to know?

Many thanks
Peter

**Comment by terual** [comment]
I guess you're using NFS to mount the music folder. Try using CIFS, there the @eaDir folder is hidden.

Anyway, this is not a LMS problem. Try googling for `hide @eaDir folder synology`

**Comment by firstWizball** [comment]
I did that google and saw that there was a pre you could set to ignore folders, but when I looked in the pref file I could not see anywhere to set it.  One of the reason I was asking here.

**Comment by terual** [comment]
See here how to exclude folders from a scan:

https://forums.slimdevices.com/forum/user-forums/logitech-media-server/90324-how-to-exclude-folders-from-a-scan?p=1021955#post1021955

**Reply by firstWizball** [reply]
I never found a prefer called ignoreDirRE in that file which is why I was asking here to see if I was missing something

**Comment by michaelherger** [comment]
> Hi the Synology nas puts a directory called @eadir in to all the music directories when you use their media server. I have looked in my pref files and I cannot see anything that says ignoredir in it. I was wondering if anyone can help set up the file o not scan for these directories.

I have a plugin called "Ignore files and folders" which you can use to exclude such folders.

**Reply by firstWizball** [reply]
thanks you
